dev/com.ibm.ws.session.cache_fat_config/fat/src/com/ibm/ws/session/cache/config/fat/SessionCacheConfigUpdateTest.java

```diff
@@ -192,11 +201,16 @@ public void testScheduleInvalidation() throws Exception {
         httpSessionCache.setScheduleInvalidationSecondHour(Integer.toString(hour2));
         httpSessionCache.setWriteFrequency("TIME_BASED_WRITE");
         httpSessionCache.setWriteInterval("15s");
-        server.setMarkToEndOfLog();
+        server.setMarkToEndOfLog(); // Only marks messages.log, does not mark the trace file
+        server.setTraceMarkToEndOfDefaultTrace();
         server.updateServerConfiguration(config);
         server.waitForConfigUpdateInLogUsingMark(APP_NAMES, EMPTY_RECYCLE_LIST);
+        String messageToCheckFor = "doScheduledInvalidation scheduled hours are " + Integer.toString(hour1) + " and " + Integer.toString(hour2);
 
-        TimeUnit.SECONDS.sleep(10); // Due to invalidation thread delay
+        // Monitor trace.log and wait for the message "doScheduledInvalidation scheduled hours are X and Y" before the test should continue
+        // This replaces the TimeUnit.SECONDS.sleep(10) used before
+        assertNotNull("Could not find Infinispan message \"" + messageToCheckFor + "\" Has the Infinispan message changed?",
```

Comment:
This is actually not an Infinispan or Hazelcast message, but comes from com.ibm.ws.session.store.common.BackedHashMap.doScheduledInvalidation(). We should update the comment here and in the Infinispan bucket to reflect this. 
`Has the logging in com.ibm.ws.session.store.common.BackedHashMap.doScheduledInvalidation() changed?`

Reply:
Thanks for hunting down the specific origin of that message Nate!  I have added that to the code in both places.